{"uuid":"d88fccdf-cdab-4a59-8b33-f91b6bca3aba","historyId":"83aa667aa10e17cc1b109e774a0c3df8","fullName":"DTransactionsTests.postTransactionWithoutDetailValue","labels":[{"name":"package","value":"DTransactionsTests"},{"name":"testClass","value":"DTransactionsTests"},{"name":"testMethod","value":"postTransactionWithoutDetailValue"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"RestAssuredTests"},{"name":"subSuite","value":"DTransactionsTests"},{"name":"host","value":"MARU"},{"name":"thread","value":"13988@MARU.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"tag","value":"SMOKE"},{"name":"severity","value":"normal"}],"links":[],"name":"postTransactionWithoutDetailValue","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertTrue(Assert.java:56)\r\n\tat org.testng.Assert.assertTrue(Assert.java:66)\r\n\tat DTransactionsTests.postTransactionWithoutDetailValue(DTransactionsTests.java:123)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:806)\r\n\tat org.testng.TestRunner.run(TestRunner.java:601)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:433)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","description":"Fail add new  transaction - Detail missing","steps":[],"attachments":[],"parameters":[],"start":1668106417317,"stop":1668106419099}